Tests/Particles/GhostsAndVirtuals/main.cpp

```diff
@@ -60,8 +60,9 @@ void test_ghosts_and_virtuals (TestParams& parms)
     Vector<BoxArray> ba(nlevs);
     ba[0].define(domain);
 
+    int n_fine = parms.nx;
     for (int lev = 1; lev < nlevs; lev++) {
-        int n_fine = parms.nx*rr[lev-1];
+        n_fine = rr[lev-1];
```

Comment:
Comparing with previous value and below, should this read:
```suggestion
        n_fine *= rr[lev-1];
```
?

Reply:
It's Friday afternoon, so all I can offer is pattern matching :D